apps/roam/src/components/settings/Settings.tsx

```diff
@@ -93,13 +93,34 @@ export const SettingsDialog = ({
       className="relative bg-white"
     >
       <div className={Classes.DIALOG_BODY}>
+        <style>{`
+          .dg-settings-tabs .bp3-tab-list {
+            overflow-y: auto;
+            max-height: 100%;
+            /* Firefox */
+            scrollbar-width: thin;
+            scrollbar-color: rgba(0, 0, 0, 0.2) transparent;
+            /* Webkit browsers */
+            &::-webkit-scrollbar {
+              width: 6px;
+            }
+            &::-webkit-scrollbar-track {
+              background: transparent;
+            }
+            &::-webkit-scrollbar-thumb {
+              background-color: rgba(0, 0, 0, 0.2);
+              border-radius: 3px;
+            }
+          }
+        `}</style>
         <Tabs
+          className="dg-settings-tabs flex h-full"
           onChange={(id) => setSelectedTabId(id)}
           selectedTabId={selectedTabId}
           vertical={true}
           renderActiveTabPanelOnly={true}
         >
-          <div className="mb-2 text-lg font-semibold text-neutral-dark">
+          <div className="pb-8 text-lg font-semibold text-neutral-dark">
```

Comment:
What was this change for?

Reply:
oh the padding of a div got in the way with the padding styling so i added it in. apparently worked on my screen size but must've broke for others. I commited a fix for this now

Reply:
https://github.com/DiscourseGraphs/discourse-graph/pull/237/commits/e18a1627a218a95d35880092015a9ade2bc5272a actually fixed in a different spot.  the `pb-8` issue is still there.